README.md

```diff
@@ -127,6 +127,60 @@ connection.connect() { result in
 }
 ```
 
+MySQLConnections should not be used to execute concurrent operations and therefore should not be shared across threads without proper synchronisation in place. It is recommended to use a connection pool if you wish to share connections between multiple threads as the connection pool will ensure your connection is not used concurrently.The example below creates a `ConnectionPool` containing a single connection and uses it to perform an insert on multiple threads:
+
+```swift
+var connectionPoolOptions = ConnectionPoolOptions.init(initialCapacity: 1, maxCapacity: 1)
+let connectionPool = MySQLConnection.createPool(host: host, user: user, password: password, database: database, port: port, characterSet: nil, connectionTimeout: 10000, poolOptions: connectionPoolOptions)
+.......
+let insertQuery = Insert(into: infos, values: "firstname", "surname", Parameter())
+let insertGroup = DispatchGroup()
+for age in 0 ... 5 {
+    insertGroup.enter()
+    connectionPool.getConnection() { connection, error in
+        guard let connection = connection else {
+            guard let error = error else {
+                print("Unable to get connection: Unknown error")
+                return insertGroup.leave()
+            }
+            print ("Unable to get connection: \(error.localizedDescription)")
+            return insertGroup.leave()
+        }
+        connection.execute(query: insertQuery, parameters: [age]) { result in
+            guard result.success else {
+                guard let error = result.asError else {
+                    print("Unable to insert age: \(age): Unknown error")
+                    return insertGroup.leave()
+                }
+                print("Unable to age: \(age): \(error.localizedDescription)")
+                return insertGroup.leave()
+            }
+            print("Succesfully inserted age: \(age)")
```

Comment:
typo -> Successfully
(and in your example output below)

Reply:
Done